docs/source/intro_accuracy_precision.rst

```diff
@@ -31,6 +31,10 @@ for when analyzing DEMs:
 .. figure:: imgs/precision_accuracy.png
     :width: 80%
 
+*Note: sometimes "accuracy" is also used to describe both types of errors, and "trueness" systematic errors, following
```

Comment:
I know it is not part of this PR, but reading the doc makes me think that we could quicly move the first paragraph of this page to the "background" section about DEMs. I would take the first sentence as it is (maybe later we can develop more about the methods used to generate DEMs), then say that a DEM is defined by:
-  a geotransform, that sets the georeferencing information: position of the top-left corner, pixel size and possible rotation. It can be obtained with `self.transform` (similar to geoutils.raster.transform).
- a projection to set the coordinates into World space. Obtained with `self.crs`.
- a nodata value to handle missing observations (`self.nodata`)
- specifically for DEMs, a vertical reference datum must be set (`self.vref`).

Reply:
Opened this suggestion in a new issue, will take care of it soon :+1: